includes/templates/admin/edit-story.php

```diff
@@ -116,6 +83,48 @@
 	),
 ];
 
+$story_path = "/web-stories/v1/$stories_rest_base/{$post->ID}/?" . build_query(
+	[
+		'_embed'           => rawurlencode(
+			implode(
+				',',
+				[ 'wp:featuredmedia', 'wp:lockuser', 'author', 'wp:publisherlogo', 'wp:term' ]
+			)
+		),
+		'context'          => 'edit',
+		'web_stories_demo' => $demo,
+		'_fields'          => rawurlencode(
+			implode(
+				',',
+				[
+					'id',
+					'title',
+					'status',
+					'slug',
+					'date',
+					'modified',
+					'excerpt',
+					'link',
+					'story_data',
+					'preview_link',
+					'edit_link',
+					'embed_post_link',
+					'permalink_template',
+					'style_presets',
+					'password',
+				]
+			)
+		),
+	]
+);
+
+if ( ! wp_validate_boolean( $demo ) ) {
+	$preload_paths[] = $story_path;
+} else {
+	$story_data             = \Google\Web_Stories\rest_preload_api_request( [], $story_path );
+	$initial_edits['story'] = ( ! empty( $story_data[ $story_path ]['body'] ) ) ? $story_data[ $story_path ]['body'] : [];
+}
```

Comment:
Observation:

I originally thought `initialEdits` would be a nice opportunity to move the demo story logic out of the REST API controller:

https://github.com/google/web-stories-wp/blob/f7a7f7a2b8159b0c4c4ff59457bca2c2792ca607/includes/REST_API/Stories_Controller.php#L81-L88

Plus we wouldn't need to make auto-drafts available via the REST API too:

https://github.com/google/web-stories-wp/blob/f7a7f7a2b8159b0c4c4ff59457bca2c2792ca607/includes/REST_API/Stories_Controller.php#L229

Similar to how `initial_edits` works in Gutenberg:

https://github.com/WordPress/wordpress-develop/blob/cb67c52be2971af8a304dbbf27e9cd78f1bfaee9/src/wp-admin/edit-form-blocks.php#L76-L96

https://github.com/WordPress/wordpress-develop/blob/cb67c52be2971af8a304dbbf27e9cd78f1bfaee9/src/wp-admin/edit-form-blocks.php#L76-L96

But now that I see it in action, it seems less possible. Hmm.

Reply:
Yeah, currently it looks like the easiest way to get demo story, maybe we will revisit this in the future, not sure about it now.